### PR TITLE
fix: New prorated personal exemption MB

### PR DESCRIPTION
**Pull request by marclafratta-humi**
https://gethumi.atlassian.net/browse/PAY-852

effective July 1 2023 19,145 TD1 basic exemption

test 24 changes
[Employee name-(EE)-PDOC-Date paid-2023-07-15.pdf](https://github.com/Humi-HR/taxman/files/11984155/Employee.name-.EE.-PDOC-Date.paid-2023-07-15.pdf)

bug 8143 spec changes
[PDOC.8143.pdf](https://github.com/Humi-HR/taxman/files/12029784/PDOC.8143.pdf)

### CODE REVIEW

File: spec/integration/taxman2023/bug_8143_spec.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@
   end
 
   it "matches PDOC's provincial tax" do
-    expect(calculate[:provincial_tax]).to be_within(0.1).of 401.87
+    expect(calculate[:provincial_tax]).to be_within(0.1).of 363.28
```
Comment (marclafratta-humi) [inline spec/integration/taxman2023/bug_8143_spec.rb:61]: Tax calculator coming back at `367.44`

Not sure which is correct, 363.28 is the PDOC amount here 
image

**Comment by claus-humi** [comment]
@marclafratta-humi You'll need to rewrite your commit messages so they will generate a release. Please look at the readme. I'll see if I can find why the test is failing.

File: spec/integration/taxman2023/bug_8143_spec.rb (hunk 1 above)
Comment (claus-humi) [reply spec/integration/taxman2023/bug_8143_spec.rb:61]: The federal is also off. Not sure what you inputted in the CPP field in PDOC (cpp max has not been reached), and the EE is EI exempt in this example?

**Review by markjconlon-humi — COMMENTED** [review]
Looks good so far. I think you need to change a commit message or make another commit prefixed with `fix:` as per readme though

**Comment by markjconlon-humi** [comment]
😢 rubocop strikes again

**Review by markjconlon-humi — APPROVED** [review]

**Pull request merged**